package.json

```diff
@@ -22,10 +22,14 @@
   "dependencies": {
     "@apollo/gateway": "^2.0.0-alpha.6",
     "@types/config": "^0.0.40",
+    "@types/morgan": "^1.9.4",
     "apollo-server": "^3.5.0",
+    "apollo-server-express": "^3.11.1",
     "config": "^3.3.6",
     "got": "^11.8.3",
     "graphql": "^16.3.0",
-    "typescript": "^4.4.4"
+    "morgan": "^1.10.0",
```

Comment:
`morgan` lib seems to be abandoned. There is no change since 2020, and it has opened issues and PRs. Are we sure we wanna use it?

Reply:
I would like to address those changes in #6, let keep this PR for refactoring, and basic log